gnome-image-installer/pages/finished/gis-finished-page.ui

```diff
@@ -131,7 +131,7 @@
                 <property name="visible">True</property>
                 <property name="can_focus">False</property>
                 <property name="margin_top">16</property>
-                <property name="label" translatable="yes">Please contact &lt;a href="mailto:support@endlessm.com"&gt;support@endlessm.com&lt;/a&gt; or join the &lt;a href="https://community.endlessos.com/"&gt;Endless Community&lt;/a&gt; to troubleshoot.</property>
+                <property name="label">Please contact %s or join the &lt;a href="https://community.endlessos.com/"&gt;Endless Community&lt;/a&gt; to troubleshoot.</property>
```

Comment:
Although perhaps this is paranoid since the real string here is going to be provided by the gtk_label_set_markup, perhaps this default label should have the "mailto:support@endlessm.com" as a placeholder rather than "%s".